flink-kubernetes-operator/pom.xml

```diff
@@ -143,6 +150,14 @@ under the License.
             <version>${junit.jupiter.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
+            <type>jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
In our current tests we spent considerable effort to avoid using mocks (and Mockito) according to the Flink codestyle: https://flink.apache.org/contributing/code-style-and-quality-common.html